### ISSUE TITLE
A11y_OSvisualupdate_MiniControGallery_GenericControls_Screenreader : Several controls in the calendar pop up are not provided with descriptive names.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/microsoft-ui-xaml#1591]
GitHubTags: #A11y_OSvisualupdate, #A11yMAS, #A11yTCS, #Win10, #DesktopApp, #WindowsUILibrary, #MAS1.3.1, #MAS4.1.2, #Narrator, #NVDA

**Environment Details:**
Application Name: OS visual update
Windows Version: Windows10
Screen Reader: Narrator, NVDA 2019.2

**Steps to Reproduce:**
1. Launch Mini Control Gallery App from the installer location.
2. Screen with all generic controls would appear on left side of the screen.
3. Verify that all the elements of "MiniControlGallery-Generic Controls" are accessible and MAS Compliant.

**Actual:**
The “Previous”, “Next” and “November 2019” buttons are not provided with descriptive names defining the actual functionality of the controls.

**Similar issue is observed in the below scenario:**
1. MiniControlGallery - Calender and Color Picker Controls -> Launch Mini Control Gallery App from the installer location. -> Screen with Calender and Color Picker controls would appear on right side of the screen.

**Expected:**
For “Previous”, “Next” and “November 2019” buttons descriptive names should be “Previous Month”, “Next Month” and “Change month” respectively which be announced to screen reader users.

**User Impact:**
If proper name is not provided for the controls, then the screen reader users will have difficulty in understanding the control and its associated functionality.

**Attachment:**
[A11y_OSvisualupdate_MiniControGallery_GenericControls_Screenreader  Several controls in the calendar pop up are not provided with descriptive names..pptx](https://github.com/microsoft/microsoft-ui-xaml/files/3830715/A11y_ControlUIupdatebasedonSantoriniOSvisualupdate_MiniControGallery_GenericControls_Screenreader.Several.controls.in.the.calendar.pop.up.are.not.provided.with.descriptive.names.pptx)

**Comment by jevansaks** [comment]
This is by design -- Next/Previous are contextual based on the "zoom level" of the control, same as "November 2019" button. We don't do much to help the sighted user here either so it seems ok.
